src/utilities/googleAnalytics.ts

```diff
@@ -1,8 +1,17 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
 import { Agents, SwapFormSubmitButtonHintStates } from "@contexts";
 import { isUndefined } from "lodash";
 import AugmentedAMM from "./augmentedAmm";
 import isBorrowing from "./isBorrowing";
 
+declare global {
+    interface Window {
+      dataLayer?: any;
+    }
+}
+
```

Comment:
lets add the sessionId in this file.

for example

const SESSION_ID = uuid();

and then we can use it in our utility for tracking. The idea being is that whenever the user refreshes the site we generate 1 session id for tracking

Reply:
added
```
import { v4 as uuidv4 } from 'uuid';

const SESSION_ID = uuidv4()
```